docs/content/guides/dagster/recommended_project_structure.mdx

```diff
@@ -0,0 +1,202 @@
+
+### For other folders
+
+#### For tests
+
+Dagster believes that testing is a fundamental part of building a maintainable data application and have integrated CI/CD processes. We recommend setting up a separate folder structure that mirrors the main project, (e.g. having a folder for test assets with any applicable subfolders) which contains the unit tests for each of the components of the data pipeline. 
+- Each of the components in Dagster such as assets, sensors and resources can all be tested separately. See more on [Testing in Dagster](https://docs.dagster.io/concepts/testing)
+- For details on [Unit Tests with stubs and mocks](/guides/dagster/transitioning-data-pipelines-from-development-to-production#advanced-unit-tests-with-stubs-and-mocks) 
+
+
+```
+project_fully_featured
+├── project_fully_featured
+├── project_fully_featured_tests
+│   ├── __init__.py
+│   ├── test_assets
+│   │   └── test_recommender
+│   │       ├── test_comment_stories.py
+│   │       ├── test_user_story_matrix.py
+│   │       └── test_user_top_recommended_stories.py
+│   │   └── test_core.py
+│   ├── test_resources
+│   │   ├── test_parquet_io_manager.py
+│   │   └── test_snowflake_io_manager.py
+│   └── test_sensors
+│       ├── __init__.py
+│       ├── test_hn_tables_updated_sensor.py
+│       └── test_slack_on_failure_sensor.py
+```
+
+
+#### For other projects outside Dagster
+
+```
+project_fully_featured
+├── dbt_project
+│  ├── README.md
+│  ├── analysis
+│  ├── config
+│  │  └── profiles.yml
+│  ├── data
+│  │  └── full_sample.csv
+│  ├── dbt_project.yml
+│  ├── macros
+│  │  ├── aggregate_actions.sql
+│  │  └── generate_schema_name.sql
+│  ├── models
+│  │  ├── activity_analytics
+│  │  │  ├── activity_daily_stats.sql
+│  │  │  ├── comment_daily_stats.sql
+│  │  │  └── story_daily_stats.sql
+│  │  ├── schema.yml
+│  │  └── sources.yml
+│  ├── snapshots
+│  ├── target
+│  │  └── manifest.json
+│  └── tests
+│      └── assert_true.sql
+├── project_fully_featured
+│ ...
+```
+
+## Other recommendations 
+
+As your data platform evolves, Dagster will enable you to orchestrate other data tools, such as [dbt projects](/integrations/dbt) or [Jupyter notebooks](/integrations/dagstermill/using-notebooks-with-dagster#using-jupyter-notebooks-with-papermill-and-dagster). 
+
+We recommend using Dagster code to integrate across various tools and projects and using a project structure to help:
+  - Teammates who are not familar with Dagster but want to leverage existing code and integrations. e.g. an analytics engineer working on a dbt project leveraging dagster for testing and orchestrating their dbt models.
+  - Dagster can help keep code in sync across so teams can work on the latest versions
```

Comment:
🤔 ❓  I like this, but I feel like it doesn't get me entirely where I need to go. A few thoughts I had when reading this:

- Structurally, how should I integrate these other tools?
- Where should I go to learn more about integrations and their best practices?